medic/medic-run.js

```diff
@@ -372,13 +372,13 @@ function main() {
         //      timeout needs to be in milliseconds, but it's
         //      given in seconds, so we multiply by 1000
         testwait.init(couchdbURI);
-        testwait.waitTestsCompleted(buildId, timeout * 1000).then(
+        testwait.waitTestsCompleted(buildId, timeout * 1000, false).then(
             function onFulfilled(value) {
-                util.medicLog("got test results");
+                util.medicLog("Successfully found test results");
                 process.exit(0);
             },
             function onRejected(error) {
-                console.error("didn't get test results: " + error);
+                console.error("Could not find test results: " + error);
```

Comment:
It will be great to suggest that a failure indicates a crashed app. Checking platform specific logs for crash reason is a good idea.